docs/resources/realm.md

```diff
@@ -132,6 +132,8 @@ The arguments below should be specified as [Go duration strings](https://golang.
 - `access_code_lifespan_user_action` - (Optional) The maximum amount of time a user has to complete login related actions, such as updating a password.
 - `action_token_generated_by_user_lifespan` - (Optional) The maximum time a user has to use a user-generated permit before it expires.
 - `action_token_generated_by_admin_lifespan` - (Optional) The maximum time a user has to use an admin-generated permit before it expires.
+- `oauth2_device_code_lifespan` - (Optional) The maximum amount of time a client has to finish the device code flow before it expires.
+- `oauth2_device_polling_interval` - (Optional) The minimum amount of time in seconds that the client should wait between polling requests to the token endpoint.
```

Comment:
So I understand why this attribute is an int, because in Keycloak it has to be specified as seconds, but this part of the docs is misleading because this is within a section that states that these attributes are duration strings.  I think we'll probably want to move this somewhere else, or allow the `oauth2_device_polling_interval` attribute to be specified as a duration string, and converted to seconds for the sake of the Keycloak API.  So, for example, you could do `oauth2_device_polling_interval = "1m"`, which would be converted to 60 seconds before it's sent to the Keycloak API.

I'm fine with either approach, but I don't think we can leave this as-is.